src/main/kotlin/no/nav/syfo/identhendelse/IdenthendelseService.kt

```diff
@@ -16,16 +17,16 @@ class IdenthendelseService(
 ) {
     private val log: Logger = LoggerFactory.getLogger(IdenthendelseService::class.java)
 
-    suspend fun handle(identhendelse: KafkaIdenthendelseDTO) {
+    fun handle(identhendelse: KafkaIdenthendelseDTO) {
         if (identhendelse.folkeregisterIdenter.size > 1) {
             val activeIdent = identhendelse.getActivePersonident()
             if (activeIdent != null) {
                 val inactiveIdenter = identhendelse.getInactivePersonidenter()
-                val aktivitetskravWithInactiveIdent = inactiveIdenter.flatMap { database.getAktivitetskrav(it) }
+                val oldPersonIdentList = inactiveIdenter.flatMap { personIdent -> database.getAktivitetskrav(personIdent).map { it.personIdent } }
```

Comment:
Er det lurt å gjøre noe form for toSet() her for å unngå duplikater i lista?

Reply:
Ja, ikke dumt tenkt. Ettersom dette er en tabell der du kan ha mange rader per personnr 💡